testsuite/forge/src/runner.rs

```diff
@@ -2,13 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::*;
-use rand::SeedableRng;
+use rand::{Rng, SeedableRng};
 use std::{
     io::{self, Write},
     process,
 };
 use structopt::{clap::arg_enum, StructOpt};
 use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, WriteColor};
+// TODO going to remove random seed once cluster deployment supports re-run genesis
+use rand::rngs::OsRng;
```

Comment:
Whats the eta on this? is this something that works now that Rustie has made changes to the fgi script?

Reply:
previously we did use 
```
let mut rng = ::rand::rngs::StdRng::from_seed([0; 32]);
CoreContext::from_rng(&mut rng)
```  
to initialize CoreContext, but I changed pure random seed instead to allow us re-run test on same cluster without wipe out existing data to avoid address conflicts.
I think Rusite already wrapped up his part and I am going to look into some potential related changes and TODOs now